### PR TITLE
Cleans up embedding link a bit.

### PR DESCRIPTION
**Pull request by AnturK**

### CODE REVIEW

File: code/modules/mob/living/carbon/human/human.dm [hunk 1]
```diff
@@ -231,11 +231,12 @@
 
 /mob/living/carbon/human/Topic(href, href_list)
 	if(usr.canUseTopic(src, BE_CLOSE, NO_DEXTERY))
-
 		if(href_list["embedded_object"])
-			var/obj/item/I = locate(href_list["embedded_object"])
-			var/obj/item/bodypart/L = locate(href_list["embedded_limb"])
-			if(!I || !L || I.loc != src || !(I in L.embedded_objects)) //no item, no limb, or item is not in limb or in the person anymore
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/human.dm]: >anturk can't read this

babbu tbh.

Comment (AnturK) [inline code/modules/mob/living/carbon/human/human.dm]: Are you implying unbound locate is good ?

Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/human.dm]: didn't say that.
just that you're a babbu.

(also it's just as safe as the rest of hrefs (read: none of it is safe))

Comment (optimumtact) [inline code/modules/mob/living/carbon/human/human.dm]: ?

it's much better to do a scoped locate than an unscoped one

Comment (RemieRichards) [inline code/modules/mob/living/carbon/human/human.dm]: @optimumtact as I said, I didn't say an unscoped locate is good.

**Pull request merged**